bin/console

```diff
@@ -1,27 +1,16 @@
 #!/usr/bin/env php
 <?php
 
-use Symfony\Bundle\FrameworkBundle\Console\Application;
+use eLife\Annotations\AppKernel;
 use Symfony\Component\Console\Input\ArgvInput;
-use Symfony\Component\Debug\Debug;
 
-// if you don't want to setup permissions the proper way, just uncomment the following PHP line
-// read https://symfony.com/doc/current/setup.html#checking-symfony-application-configuration-and-setup
-// for more information
-//umask(0000);
+require_once __DIR__.'/../vendor/autoload.php';
 
+umask(002);
```

Comment:
it doesn't make a difference for this particular number, but umasks are octal so should be `0002` for clarity (leading `0` indicates it's an octal number, then `002` is `rw-rw-r--`)